.github/workflows/doctor.yml

```diff
@@ -0,0 +1,31 @@
+name: Dr. Releng
+
+on:
+  pull_request:
+    paths:
+      - policy/templates/**
+
+jobs:
+  doctor:
+    runs-on: ubuntu-latest
+    container: tykio/gromit:v1.4.6-rc1
+    strategy:
+      fail-fast: false
+      matrix:
+        repo: [tyk, tyk-analytics, tyk-pump, tyk-identity-broker, tyk-sink, portal]
+        branch: [master]
+        bundle: [releng, sync]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: TykTechnologies/${{ matrix.repo }}
+          token: ${{ secrets.ORG_GH_TOKEN }}
+          path: ${{ matrix.repo }}
+
+      - name: Generate
+        run: gromit bundle --bundle ${{ matrix.bundle }} --repo ${{ matrix.repo }} gen ${{ matrix.repo }}
+
+      - name: Check for zero diffs
+        id: diff
+        run: gromit git diff ${{ matrix.repo }}
```

Comment:
gromit git diff won't show any output here, because files has not been staged

Reply:
diff now unifies cached and uncached.

Reply:
diff now unifies cached and uncached.